docs/cosmos-hub/installation.md

```diff
@@ -0,0 +1,74 @@
+## Install Gaia
+
+This guide will explain how to install the `gaiad` and `gaiacli` entrypoints onto your system. With these installed on a server, you can participate in the mainnet as either a [Full Node](./join-mainnet.md) or a [Validator](./validators/validator-setup.md).
+
+### Install Go
+
+Install `go` by following the [official docs](https://golang.org/doc/install). Remember to set your `$GOPATH` and `$PATH` environment variables, for example:
+
+```bash
+mkdir -p $HOME/go/bin
+echo "export GOPATH=$HOME/go" >> ~/.bash_profile
+echo "export PATH=\$PATH:\$GOPATH/bin" >> ~/.bash_profile
+source ~/.bash_profile
+```
+
+::: tip
+**Go 1.12+** is required for the Cosmos SDK.
+:::
+
+### Install the binaries
+
+Next, let's install the latest version of Gaia. Make sure you `git checkout` the correct [released version](https://github.com/cosmos/gaia/releases).
+
+::: warning
+For the mainnet, make sure your version is greater than `v0.33.0`
+:::
+
+::: warning
+Older releases up to 0.34.x are available from the [Cosmos SDK releases page](https://github.com/cosmos/cosmos-sdk/releases).
+::: 
+
+
+```bash
+mkdir -p $GOPATH/src/github.com/cosmos
+cd $GOPATH/src/github.com/cosmos
+git clone -b <latest-release-tag> https://github.com/cosmos/gaia
+cd cosmos-sdk && make tools install
+```
+
+> *NOTE*: If you have issues at this step, please check that you have the latest stable version of GO installed.
+
+That will install the `gaiad` and `gaiacli` binaries. Verify that everything is OK:
+
+```bash
+$ gaiad version --long
+$ gaiacli version --long
+```
+
+`gaiacli` for instance should output something similar to:
+
+```
+cosmos-sdk: 0.33.0
```

Comment:
not related, but should the version also print the gaia release version ? @alessio